cfme/common/provider.py

```diff
@@ -194,15 +196,18 @@ def create(self, cancel=False, validate_credentials=True, check_existing=False,
                         # filling credentials
                         if hasattr(endpoint, 'credentials'):
                             endp_view.fill(endpoint.credentials.view_value_mapping)
-
-                            if validate_credentials and hasattr(endp_view, 'validate'):
-                                # there are some endpoints which don't demand validation like
-                                #  RSA key pair
-                                endp_view.validate.click()
+                        # sometimes we have cases that we need to validate even though
+                        # there is no credentials, such as Hawkular endpoint
+                        if validate_credentials and hasattr(endp_view, 'validate'):
+                            # there are some endpoints which don't demand validation like
+                            #  RSA key pair
+                            endp_view.validate.click()
                 if self.one_of(InfraProvider):
                     main_view_obj = InfraProvidersView
                 elif self.one_of(CloudProvider):
                     main_view_obj = CloudProvidersView
+                elif self.one_of(ContainersProvider):
```

Comment:
we will need to get rid of these statements. I'll do that when we finish conversion for all providers